README.md

```diff
@@ -103,8 +103,7 @@ This is enforced for any case, whether a label matcher is specified in the origi
 
 Similar to query endpoint, for metadata endpoints `/api/v1/series`, `/api/v1/labels`, `/api/v1/label/<name>/values` the proxy injects the specified label all the provided `match[]` selectors.
 
-NOTE: At the moment of creation `/api/v1/labels`, `/api/v1/label/<name>/values` does not support `match[]` so they are disabled by default. Use `-enable-label-apis` flag to enable
-those (see https://github.com/prometheus/prometheus/issues/6178 for tracking development).
+NOTE: When the `/api/v1/labels` and `/api/v1/label/<name>/values` endpoints were added to `prom-label-proxy`, the Prometheus and Thanos endpoints didn't support the `match[]` parameter hence the `prom-label-proxy` labels endpoints are disabled by default. Use the `-enable-label-apis` flag to enable after you've ensured that the upstream endpoints support label selectors (see https://github.com/prometheus/prometheus/issues/6178 and https://github.com/thanos-io/thanos/issues/3351).
```

Comment:
Thanos supports label matchers already in https://github.com/thanos-io/thanos/pull/3566, just implementing it in a not so efficient way (via Series).
Now efficient labels matchers pushdown is supported too in the latest Thanos release (v0.23.0)

Reply:
Thanks! Do you recommend that we mention Thanos >= v0.23.0 explicitly here?

Reply:
ping @yeya24 :))

Reply:
Seems like the the feature was added in [Thanos v0.18.0](https://github.com/thanos-io/thanos/releases/tag/v0.18.0)

Reply:
I've clarified the text, mentioning version numbers instead of pull requests IDs.